packages/testsuite/cypress/e2e/transaction/transactions-configuration-subsystem-transaction-jdbc.cy.ts

```diff
@@ -87,6 +87,9 @@ describe("TESTS: Configuration => Sybsystem => Transaction => Path", () => {
       //When use-jdbc-store toggled to true, it needs jdbc-datasource to be set
       if (!defaultValue) {
         cy.text(jdbcForm, jdbcStoreDatasource, nonEmptyDS.name);
+        // the value need to be picked from the suggestions. this is not a dropdown menu.
```

Comment:
I think it would be better to simply click somewhere else, or press Tab (or some other relevant button) to change focus and let the suggestions dismiss.

Reply:
I think we could create method similar to `text` method which could be named `textWithSuggestions` and will handle the suggestions. WDYT?

Reply:
I suggest to make the detection and getting rid of the suggestion and plug it inside this method.

Reply:
For example, see Manatoko: https://github.com/hal/manatoko/blob/5a78e23b0477ef63859c8be4e96859b4f80e6ea8/ui/src/main/java/org/jboss/hal/testsuite/fragment/FormFragment.java#L182

Reply:
Thank you for inspiration from another TS. I used function `blur` which cause focus lose from current input and suggestion are closed. https://docs.cypress.io/api/commands/blur

It seems this helped also in another components tests (core-management, runtime, transaction):
before fix: 
https://github.com/hal/berg/actions/runs/6764990344
with fix:
https://github.com/kstekovi/berg/actions/runs/6770941850/job/18400353034